### ISSUE TITLE
Allow sections in recents list to be reordered

### ISSUE THREAD

**Issue by ghost** [element-hq/element-web#2536]
Right now the order of sections is fixed at `Favourites`, `People`, `Rooms` and `Low Priority`.

**Comment by ara4n** [comment]
Yup. This is very closely related to #1373 and will likely happen at the same time.

**Comment by t3chguy** [comment]
No longer relevant in the new room list
